.travis.yml

```diff
@@ -3,15 +3,18 @@ sudo: false
 cache:
   cargo
 rust:
-  - 1.20.0
+  - 1.24.0
```

Comment:
this bump is required to do the new travis stuff; we'll be releasing a new version anyway, so this is fine